docs/navigation-configuration.md

```diff
@@ -1,45 +1,41 @@
-# Navigation configuration
+#  Basic navigation configuration
 
 Read these guides to get started with configuring your navigation:
 
-* [Navigation elements](#navigation-elements) 
+* [Navigation elements](#navigation-elements)
 * [First steps](#first-steps)
 * [Basic navigation parameters](#basic-navigation-parameters)
 * [Grouping navigation nodes](#grouping-navigation-nodes)
   * [category](#category)
   * [viewGroup](#viewgroup)
-* [Creating a dynamic path](#creating-a-dynamic-path)
-  * [pathSegment parameters](#pathsegment-parameters)
-  * [viewUrl parameters](#viewurl-parameters)
-  * [Node parameters](#node-parameters)
-  * [Dynamic viewUrl](#dynamic-viewurl)
 
 If you are already familiar with the basics, take a look at:
+* [Advanced navigation configuration](navigation-advanced.md)
 * [Full reference list of navigation parameters](navigation-parameters-reference.md)
 
 ## Navigation elements
 
-There are three main elements to Luigi: 
+There are three main elements to Luigi:
 
-1. Top navigation which displays the main navigation path. Context, product, app, and profile [switchers](navigation-parameters-reference.md/#context-switcher) can also be displayed here. 
-2. Side navigation which displays the child nodes of the root navigation node. It can include structures like collapsible dropdowns and menus. 
+1. Top navigation which displays the main navigation path. Context, product, app, and profile [switchers](navigation-parameters-reference.md/#context-switcher) can also be displayed here.
```

Comment:
```suggestion
1. Top navigation which displays the main navigation path. Context, product, app, and profile [switchers](navigation-parameters-reference.md#context-switcher) can also be displayed here.
```
Removing the slash after .md since it is not valid url (even tough it's working because of github).